react_week2_fixed/14.md

```diff
@@ -0,0 +1,35 @@
+## Objects Advanced
+
+Now that we know how Objects look like in JavaScript, let's constuct our own objects. 
+
+``` JavaScript
+let person = {
+  name: "John", 
+  age: 10, 
+  hometown: "davis"
+}
+```
+
+This method only allows you to create one object. If we want to create several objects, we will want to create an object type. 
+
+```javascript
+function person(name, age, hometown){
```

Comment:
Can you make sure you use the button to demonstrate the objects. 
For example, 

let person = {
   name: "Potato", 
   bounce_time: 10, 
   shape: "square"
    }